### PR TITLE
Add Hedera networks

### PR DESCRIPTION
**Pull request by Neurone**

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
explorer standard must be 'none' or 'EIP3091'

**Comment by Neurone** [comment]
> explorer standard must be 'none' or 'EIP3091'

Ops, sorry for that. Fixed.

**Review by ligi — CHANGES_REQUESTED** [review]
please remove the ens fields

**Comment by Neurone** [comment]
> please remove the ens fields

Done!

File: _data/chains/eip155-297.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "infoURL": "https://hedera.com",
+  "shortName": "hedera-previewnet",
+  "chainId": 297,
+  "networkId": 297,
+  "slip44": 3030,
+  "explorers": [
+    {
+      "name": "HashScan",
+      "url": "https://hashscan.io/previewnet",
```
Comment (ligi) [inline _data/chains/eip155-297.json:21]: does not seem to be reachable

File: _data/chains/eip155-296.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  "infoURL": "https://hedera.com",
+  "shortName": "hedera-testnet",
+  "chainId": 296,
+  "networkId": 296,
+  "slip44": 3030,
+  "explorers": [
+    {
+      "name": "HashScan",
+      "url": "https://hashscan.io/testnet",
```
Comment (ligi) [inline _data/chains/eip155-296.json:21]: is 404ing

File: _data/chains/eip155-295.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  "infoURL": "https://hedera.com",
+  "shortName": "hedera-mainnet",
+  "chainId": 295,
+  "networkId": 295,
+  "slip44": 3030,
+  "explorers": [
+    {
+      "name": "HashScan",
+      "url": "https://hashscan.io/mainnet",
```
Comment (ligi) [inline _data/chains/eip155-295.json:21]: 404ing

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-296.json (hunk 2 above)
Comment (Neurone) [reply _data/chains/eip155-296.json:21]: I set the url for the API, I thought it was needed that format reading the EIP3091 specification. You can get results using for example the url: https://hashscan.io/testnet/block/1000

You can get the dashboard homepage with this url: https://hashscan.io/testnet/dashboard 

Should I insert that instead?

File: _data/chains/eip155-295.json (hunk 3 above)
Comment (Neurone) [reply _data/chains/eip155-295.json:21]: Same comment as above

File: _data/chains/eip155-297.json (hunk 1 above)
Comment (Neurone) [reply _data/chains/eip155-297.json:21]: same comment as below

File: _data/chains/eip155-296.json (hunk 2 above)
Comment (ligi) [reply _data/chains/eip155-296.json:21]: I think https://hashscan.io/testnet should redirect to https://hashscan.io/testnet/dashboard

Comment (Neurone) [reply _data/chains/eip155-296.json:21]: Ok, understood, I'll suggest them implementing that redirect.

In the meanwhile, to progress the PR and if it is ok for you, I'm going to update the URLs with the "human paths", and set `none` as supported standard. Once they implement the redirect, I will submit a PR updating the URLs and the supported standard.

Does it work for you?

Comment (ligi) [reply _data/chains/eip155-296.json:21]: yes

**Review by ligi — APPROVED** [review]

**Pull request merged**